### ISSUE TITLE
Edit high res?

### ISSUE THREAD

**Issue by cian0** [sail-sg/EditAnything#8]
Similar to high res inpainting in SD webui, would this be possible?

**Comment by gasvn** [comment]
Thanks. Can you give me a link to what you said? That's possible if we train the model with higher resolution.

**Comment by cian0** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features


I don't think there's a need to train with a higher resolution for this as it's simply a technique to resize and repaint the masked area. This is usually used for faces in SD

Under inpaint area, the doc says:

Inpaint area
Normally, inpainting resizes the image to the target resolution specified in the UI. With Inpaint area: Only masked enabled, only the masked region is resized, and after processing it is pasted back to the original picture. This allows you to work with large pictures and render the inpainted object at a much larger resolution.

**Comment by gasvn** [comment]
I understand it now. This would be a good feature. We can only edit a part of the project in high resolution.

**Comment by cian0** [comment]
Yup though I'm not sure if I get why outputs would have blurred faces as per documentation..

**Comment by gasvn** [comment]
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/20515144/231177441-223db9b8-0368-4192-ba8e-ea66f34a602c.png">
Here are some images from the SAM dataset. You can find more at https://segment-anything.com/dataset/index.html
We are solving this problem by finetuning models on the Laion dataset.

**Comment by kadirnar** [comment]
> Similar to high res inpainting in SD webui, would this be possible?

I want to add this feature to my demo application. But I couldn't find the codes. Can you help me?
Demo: https://huggingface.co/spaces/ArtGAN/Stable-Diffusion-ControlNet-WebUI

**Comment by cian0** [comment]
> > Similar to high res inpainting in SD webui, would this be possible?
> 
> I want to add this feature to my demo application. But I couldn't find the codes. Can you help me? Demo: https://huggingface.co/spaces/ArtGAN/Stable-Diffusion-ControlNet-WebUI

I believe it's here :

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/22bcc7be428c94e9408f589966c2040187245d81/modules/processing.py#L985
